util/macros.go

```diff
@@ -0,0 +1,48 @@
+package util
```

Comment:
Lets not create a "util" package ... nobody will know what to expect inside of it, and nobody will know how to find the code inside if they weren't already familiar with it.

Maybe a "macros" or "templates" package?

Reply:
@hhhjort Sure. Makes sense! I will update the PR.

Reply:
@hhhjort moved macros to it's own package and added some unit tests for it.

Reply:
May be worth reading this someday: https://blog.golang.org/package-names

It mentions this... but also has lots of other advice.